controls/1_4_secure_boot_settings.rb

```diff
@@ -17,6 +17,8 @@
 
 title '1.4 Secure Boot Settings'
 
+grubconflocations = inspec.profile.file('grub_conf_locations').split
```

Comment:
+1 I agree on the extraction of the grub values. I am not 100% sure if we should use `inspec.profile.file('grub_conf_locations')`, since I prefer to use that pattern for data loading like yaml files.

For readability, I recommend we use a custom resource as we use for [ssh](https://github.com/dev-sec/ssh-baseline/blob/master/libraries/ssh_crypto.rb#L19):

```ruby
class GrubConf < Inspec.resource(1) # rubocop:disable Metrics/ClassLength
  name 'grub_conf'

  def locations
    %w(/boot/grub/grub.conf /boot/grub/grub.cfg /boot/grub/menu.lst /boot/boot/grub/grub.conf /boot/boot/grub/grub.cfg /boot/boot/grub/menu.lst /boot/grub2/grub.cfg)
  end
end
``` 

```ruby
# a control would look like:
  describe.one do
     grub_conf. locations.each do |f|
       describe file(f) do
```

Reply:
Sure. I'll revisit this in a few days when I'm back with the client.